src/components/Composer/index.js

```diff
@@ -172,6 +173,16 @@ export class Composer extends Component {
 		replaceCaret(el);
 	}
 
+	componentWillMount() {
+		if(this.props.notifyEmojiSelect){
+			this.props.notifyEmojiSelect(this.handleNotifyEmojiSelect);
+		}
+	}
+
+	handleNotifyEmojiSelect(emojiColonName) {
+		this.el.innerText = this.el.innerText + emojiColonName;
```

Comment:
```suggestion
		this.el.innerText = `${ this.el.innerText } emojiColonName`;
```

Reply:
Incorporated this change in lastest commit :+1: thanks for the feedback :smiley: